src/runner/__tests__/runners.ts

```diff
@@ -177,3 +178,14 @@ describe('Additional JavaScript features are not available in Source Native', ()
     }
   )
 })
+
+// HTML Unit Tests
+
+test('Error handling script is injected in HTML code', async () => {
+  const htmlDocument: string = '<p>Hello World!</p>'
+  const htmlContext: Context = mockContext(Chapter.HTML, Variant.DEFAULT)
+  const result = await runInContext(htmlDocument, htmlContext)
+
+  expect(result.status).toStrictEqual('finished')
+  expect((result as any).value).toStrictEqual(ERROR_HANDLING_SCRIPT + htmlDocument)
```

Comment:
```suggestion
  expect((result as Finished).value).toStrictEqual(ERROR_HANDLING_SCRIPT + htmlDocument)
```

Let's avoid using `any`. In fact, we should not even be performing any type casts because once we introduce a type cast into the codebase, we lose any guarantee of type safety. Ideally, we would make use of [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) to avoid having to type cast, but I understand that this would be rather out of scope for this PR + the entire codebase unfortunately uses type casts everywhere.